src/frameworks/TYPO3.md

```diff
@@ -0,0 +1,17 @@
+# Getting started
```

Comment:
This file should be named `typo3.md`, not `TYPO3`.  All files/paths are lower-case.  (No one calls it `drupal` either, but consistency... :smile: )